platforms/corda-ent/configuration/roles/setup/nmap/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+##############################################################################################
+# This role creates the value file for nmap
+##############################################################################################
+
+# create deployment file for CENM nmap service
+- name: "create nmap"
+  include_role:
+    name: helm_component
+  vars:
+    component_name: "{{ org.name | lower }}nmap"
+    component_ns: "{{ org.name | lower }}-ent"
+    type: "nmap"
+    values_dir: "{{playbook_dir}}/../../../{{ org.gitops.release_dir }}"
+    name: "{{ org.name | lower }}"
+    charts_dir: "{{ org.gitops.chart_source }}"
+    notary_name: "{{ org.services.notary.name }}"
+    git_url: "{{ org.gitops.git_ssh }}"
+    git_branch: "{{ org.gitops.branch }}"
+    node_name: "{{ org.services.networkmap.name }}"
+    storageclass: "awsstorageclass"
+    init_container_name: "{{ network.docker.url }}/alpine-utils:1.0"
+    image_pull_secret: "regcred"
+    nmap_image_name: corda/enterprise-networkmap
+    nmap_image_tag: "1.2-zulu-openjdk8u242"
+    vault_addr: "{{ org.vault.url }}"
+    vault_cert_secret_prefix: "secret/cenm"
+    external_url_suffix: "{{ org.external_url_suffix }}"
+    nmap_port: "{{ org.services.networkmap.ports }}"
+    ssh_username: "{{ org.services.networkmap.name }}"
+    ssh_password: "{{ org.services.networkmap.name }}P"
```

Comment:
Why to attach anything in password rather take whole thing from user.

Reply:
For this we will create an issue to manage the passwords of all the cenm services

Reply:
ok.

Reply:
for reference #853